protokube/pkg/protokube/aws_volume.go

```diff
@@ -164,73 +164,83 @@ func newEc2Filter(name string, value string) *ec2.Filter {
 func (a *AWSVolumes) findVolumes(request *ec2.DescribeVolumesInput) ([]*Volume, error) {
```

Comment:
We have "k8s.io/etcd/events" and "k8s.io/etcd/main" tag keys.  Not sure how we can filter on two tags.  Should we make two calls?